emission/core/wrapper/suggestion_sys.py

```diff
@@ -105,8 +105,17 @@ def search(api_key, term, location):
     }
     return request(API_HOST, SEARCH_PATH, api_key, url_params=url_params)
 
+def newsearch(api_key, lat, lon, radius):
+    url_params = {
+        'latitude': lat,
+        'longitude': lon,
+        'radius' : radius,
+        'limit': SEARCH_LIMIT
+    }
+    return request(API_HOST, SEARCH_PATH, api_key, url_params=url_params)
+
 """
-YELP API: Function to retrieve all reviews related to the business. 
+YELP API: Function to retrieve all reviews related to the business.
```

Comment:
Can you not make unrelated whitespace changes? It makes it harder to see what changes are included in your PR.